duvm/assert_randomize.py

```diff
@@ -17,12 +17,12 @@ class AssertRandomize(filters.LineListener):
                     filters.UVCLineBroadcaster]
 
     randomize_re = re.compile("(((\.)|(std::))randomize)\(")
-    assert_re = re.compile("^\s*(//)*\s*`cmn_assert\(")
+    assert_re = re.compile("^\s*(//)*\s*`cmn_f?assert\(")
 
     def _update(self, line_no, line):
         if self.randomize_re.search(line):
             if not self.assert_re.search(line):
-                self.error(line_no, line, "Randomize call was not wrapped with a `cmn_assert.")
+                self.error(line_no, line, "Randomize call was not wrapped with a `cmn_assert|`cmn_fassert.")
```

Comment:
Could you change this to be:
```suggestion
                self.error(line_no, line, "Randomize call was not wrapped with a `cmn_assert or a `cmn_fassert.")
```

I think it's a little more readable for a user seeing this message.